tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -91,7 +90,7 @@ def _set_thread_name(name: str):
 #
 # Possible values for the collect_logs parameter
 #
-class CollectLogs(Enum):
```

Comment:
comparing a string to one of the enum items was failing; changed this to be a class with string data members instead of an Enum